lumicks/pylake/tests/test_imaging_confocal/test_kymo.py

```diff
@@ -972,3 +973,60 @@ def test_plot_with_lf_force():
 
     with pytest.raises(RuntimeError, match="Desired force channel 1x not available in h5 file"):
         kymo.plot_with_force("1x", "red")
+
+
+@cleanup
```

Comment:
same thing here with separating out the plotting tests? `test_kymo_plotting.py`?